qiskit_experiments/characterization/t1_analysis.py

```diff
@@ -159,7 +161,16 @@ def fit_fun(x, a, tau, c):
         else:
             figures = None
 
-        return [analysis_result], figures
+        res_v1 = DbAnalysisResultV1(
```

Comment:
This looks like boiler plate code that will need to be copied to each experiment without much change. Can it be moved to either part the service or BaseAnalysis classes so that we don't need to do it for every subclass, and then `_run_analysis` will just return the list of dicts and figures?